packages/frameworks/src/frameworks.ts

```diff
@@ -104,6 +104,7 @@ export const frameworks = [
     devCommand: 'next dev --port $PORT',
     buildCommand: 'next build',
     getOutputDirName: async () => 'public',
+    cachePattern: '.next/cache/**',
```

Comment:
Isn't this handled in the runtime in `prepareCache` already or do we need it here as well?

Reply:
It is but we need a way to document it and this pattern like we have for the other frameworks here